src/textual/app.py

```diff
@@ -1753,20 +1753,19 @@ def _load_screen_css(self, screen: Screen):
 
         update = False
         for path in screen.css_path:
-            if not self.stylesheet.has_source(path):
+            if not self.stylesheet.has_source((str(path), "")):
```

Comment:
I'd like to veto the tuple here. It doesn't read well. Could the second value be a keyword arg?

Reply:
Yeah, that sounds good. See 6c6eecf.